



<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1-strict.dtd ">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Arch Linux Repositories [packages] View of /gcc/repos/core-i686/buildfix_new_bison.patch</title>
<meta http-equiv="content-type" content="text/html; charset=utf-8" />
<link rel="stylesheet" href="http://archlinux.org/media/arch.css" />
<link rel="stylesheet" href="/static/styles.css" type="text/css" />

<link rel="icon" href="http://archlinux.org/media/favicon.ico" type="image/x-icon" />
<link rel="shortcut icon" href="http://archlinux.org/media/favicon.ico" type="image/x-icon" />
<style type="text/css">
div.searchbar {
font-size: 13px;
font-weight: bold;
position: absolute;
right: 0.2em;
top: 152px;
}
</style>
</head>
<body>
<div id="head_container">
<div id="title">
<div id="logo"><h1 id="archtitle"><a href="/" title="Arch Linux (Home)">Arch Linux</a></h1></div>
</div>
<div id="main_nav">
<ul>
<li><a href="http://archlinux.org/download/">Get Arch</a></li>
<li><a href="http://aur.archlinux.org">AUR</a></li>
<li><a href="http://bugs.archlinux.org">Bugs</a></li>
<li><a href="http://wiki.archlinux.org">Wiki</a></li>
<li><a href="http://bbs.archlinux.org">Forums</a></li>
<li><a href="http://archlinux.org">Home</a></li>
</ul>
</div>
<div id="ads">
<script type="text/javascript"><!--
google_ad_client = "pub-3170555743375154";
google_ad_width = 468;
google_ad_height = 60;
google_ad_format = "468x60_as";
google_color_border = "ffffff";
google_color_bg = "ffffff";
google_color_link = "0771A6";
google_color_url = "99AACC";
google_color_text = "000000";
//--></script>
<script type="text/javascript" src="http://pagead2.googlesyndication.com/pagead/show_ads.js"></script>
</div>
</div>
<div id="content">
<div class="vc_navheader">

<form method="get" action="/viewvc.cgi/">

<div>
<strong>

<a href="/viewvc.cgi/">

[packages]</a>
/

<a href="/viewvc.cgi/gcc/">

gcc</a>
/

<a href="/viewvc.cgi/gcc/repos/">

repos</a>
/

<a href="/viewvc.cgi/gcc/repos/core-i686/">

core-i686</a>
/

<a href="/viewvc.cgi/gcc/repos/core-i686/buildfix_new_bison.patch?view=log">

buildfix_new_bison.patch</a>


</strong>

</div>
<div class="searchbar">


<strong>Repository:</strong>
<select name="root" onchange="submit()">


<option value="*viewroots*">Repository Listing</option>












































<optgroup label="CVS Repositories"><option>community</option><option>core</option><option>extra</option><option>unstable</option></optgroup>


<optgroup label="Subversion Repositories"><option selected="selected">packages</option></optgroup>

</select>
<input type="submit" value="Go" />

</div>

</form>

</div>
<h1>View of /gcc/repos/core-i686/buildfix_new_bison.patch</h1>

<p style="margin:0;">

<a href="/viewvc.cgi/gcc/repos/core-i686/"><img src="/static/images/back_small.png" width="16" height="16" alt="Parent Directory" /> Parent Directory</a>

| <a href="/viewvc.cgi/gcc/repos/core-i686/buildfix_new_bison.patch?view=log#rev19322"><img src="/static/images/log.png" width="16" height="16" alt="Revision Log" /> Revision Log</a>




</p>

<hr />
<div class="vc_summary">
Revision <a href="/viewvc.cgi?view=rev&amp;revision=19322"><strong>19322</strong></a> -
(<a href="/viewvc.cgi/gcc/repos/core-i686/buildfix_new_bison.patch?revision=19322"><strong>download</strong></a>)

(<a href="/viewvc.cgi/gcc/repos/core-i686/buildfix_new_bison.patch?annotate=19322"><strong>annotate</strong></a>)

<br /><em>Thu Nov 20 11:36:53 2008 UTC</em>
(6 months ago)
by <em>andyrtr</em>

<br />Original Path: <a href="/viewvc.cgi/gcc/trunk/buildfix_new_bison.patch?view=log&amp;pathrev=19322"><em>gcc/trunk/buildfix_new_bison.patch</em></a>






<br />File size: 423 byte(s)



<pre class="vc_log">upgpkg: gcc 4.3.2-2
    rebuild for new binutils/glibc</pre>

</div>
<div id="vc_markup"><pre><PRE>
--- gcc/treelang/parse.y	2007-07-18 08:19:29.000000000 +0000
+++ gcc/treelang/parse.y.new	2008-11-20 06:40:00.013275486 +0000
@@ -369,7 +369,6 @@
   current_function = NULL;
 }
 ;
-
 variable_defs_opt:
 /* Nil.  */ {
   $$ = 0;
@@ -685,9 +684,9 @@
 
 expressions_with_commas_opt:
 /* Nil.  */ {
-$$ = 0
+$$ = 0;
 }
-|expressions_with_commas { $$ = $1 }
+|expressions_with_commas { $$ = $1; }
 ;
 
 expressions_with_commas:
</PRE>
</pre></div>

<hr />
<div class="foot">
Copyright &copy; 2002-2008
<a href="mailto:jvinet@zeroflux.org" title="contact Judd Vinet">Judd Vinet</a> and
<a href="mailto:aaron@archlinux.org" title="contact Aaron Griffin">Aaron Griffin</a>.
<br />
The Arch Linux name and logo are recognized trademarks. Some rights reserved.
</div>
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
var pageTracker = _gat._getTracker("UA-3589020-1");
pageTracker._setDomainName("archlinux.org");
pageTracker._initData();
pageTracker._trackPageview();
</script>
</body>
</html>

